### PR TITLE
Branch code quality

### LINKED ISSUE #225: Code Quality Check

### PR DESCRIPTION
**Pull request by goatygoatygoat**
Checked code quality for storage and model components in main

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/ClearCommand.java [hunk 1]
```diff
@@ -6,13 +6,13 @@
...
 /**
- * Clears the address book.
+ * Clears the data in Teaching Assistant.
  */
 public class ClearCommand extends Command {
 
     public static final String COMMAND_WORD = "clear";
-    public static final String MESSAGE_SUCCESS = "Address book has been cleared!";
 
+    public static final String MESSAGE_SUCCESS = "Address book has been cleared!";
```
Comment (nicholastanvis) [inline src/main/java/seedu/address/logic/commands/ClearCommand.java]: heh

File: src/main/java/seedu/address/logic/Logic.java [hunk 2]
```diff
@@ -12,9 +12,10 @@
 import seedu.address.model.entry.Entry;
 
 /**
- * API of the Logic component
+ * API of the Logic component.
```
Comment (nicholastanvis) [inline src/main/java/seedu/address/logic/Logic.java:15]: um... you should only use periods when it's a sentence. things like that don't need it but it looks like you added them in for the entire pr anyway so 💀

File: src/main/java/seedu/address/model/contact/UniqueContactList.java [hunk 3]
```diff
@@ -15,12 +15,13 @@
...
- * that the contact being added or updated is
- * unique in terms of identity in the UniqueContactList. However, the removal of
- * a contact uses Contact#equals(Object) so
- * as to ensure that the contact with exactly the same fields will be removed.
+ *
+ * As such, adding and updating of contacts uses Contact#isSameContact(Contact) for equality
+ * so as to ensure that the contact being added or updated is unique in terms of identity
+ * in the UniqueContactList.
+ *
+ * However, the removal of a contact uses Contact#equals(Object) so as to
```
Comment (nicholastanvis) [inline src/main/java/seedu/address/model/contact/UniqueContactList.java]: our isSameContact uses equals, this comment is not correct

File: src/main/java/seedu/address/model/tag/Tag.java [hunk 4]
```diff
@@ -26,7 +24,7 @@ public Tag(String tagName) {
     }
 
     /**
-     * Returns true if a given string is a valid tag name.
+     * Returns true if a given string is a valid name.
```
Comment (nicholastanvis) [inline src/main/java/seedu/address/model/tag/Tag.java:27]: why lol

**Review by nicholastanvis — CHANGES_REQUESTED** [review]

File: src/main/java/seedu/address/model/tag/Tag.java (hunk 4 above)
Comment (goatygoatygoat) [reply src/main/java/seedu/address/model/tag/Tag.java:27]: I like

**Review by lrj689 — APPROVED** [review]

File: src/main/java/seedu/address/model/contact/ContactName.java [hunk 5]
```diff
@@ -14,18 +13,16 @@
             + "- consist of only alphabets, spaces and hyphens\n"
             + "- start and end with alphabets";
 
-    /*
-     * The first character of the name must not be a whitespace,
+    /**
+     * The first character of the address must not be a whitespace,
```
Comment (tsh22) [inline src/main/java/seedu/address/model/contact/ContactName.java]: Wait why is this address lol

**Review by tsh22 — APPROVED** [review]

**Review by nicholastanvis — APPROVED** [review]

File: src/main/java/seedu/address/logic/commands/ClearCommand.java (hunk 1 above)
Comment (nicholastanvis) [reply src/main/java/seedu/address/logic/commands/ClearCommand.java]: why you resolve this btw are you gonna make it say "Address book"

Comment (nicholastanvis) [reply src/main/java/seedu/address/logic/commands/ClearCommand.java]: oh read the wrong commit xd

**Pull request merged**